app/models/manageiq/providers/kubernetes/container_manager_mixin.rb

```
@@ -266,4 +266,16 @@ def evaluate_alert(_alert_id, _event)
     # and these should automatically be translated to alerts.
     true
   end
+
+  def queue_metrics_capture(interval_name = 'realtime', options = {})
+    targets = Metric::Targets.capture_container_targets([self], {})
+
+    targets.each do |target|
+      begin
+        target.perf_capture_queue(interval_name, options)
+      rescue StandardError => err
+        _log.warn("Failed to queue perf_capture for target [#{target.class.name}], [#{target.id}], [#{target.name}]: #{err}")
```

:+1: [perf_capture_queue](https://github.com/ManageIQ/manageiq/blob/ea927c7205/app/models/metric/ci_mixin/capture.rb#L32) may raise exceptions, especially ArgumentError.
However, I expect in most cases where one target raises, all would raise,
so perhaps this function should just let exception propogate?
[Your call, I dont have opinion here]

---

yeah we should let it propagate and rather do _log.error, there shouldn't be a case where only 1 fails

---

thanks @cben @Ladas 👍

now it's `_log.error` and propagating :cherry_blossom: